message AnyEnum {
  optional string type_url = 1;
  optional int32 value = 2;
}
